### DISCUSSION TITLE
Two Question and Three Issue

### DISCUSSION THREAD

**Discussion by Resonance2806** [ppy/osu#14136] [Q&A] [answered]
Question 1. When will osu!lazer finish and release as a major update for osu! ?
Question 2. Will the conversion and fun mods from osu!lazer be ranked? Because I see some of them make the game harder or easier
Issue 1: When playing osu!lazer, I often got lag spike both in the main menu and during the game, game freeze but music not, after that the game resume but I will miss a lot or fail because I can't see click the note during freeze
Issue 2: If the game is in full-screen mode and alt-tabbed away from the game then coming back, osu!lazer will freeze, music stop playing, then I had to close the game manually and restarted it in order to play the game
Issue 3: Some skin cannot be imported and rolled back in osu!lazer
I put both the issue and question here because I don't want to create another topic for issue reporting

**Comment by bdach** [comment, marked as answer]
> Question 1. When will osu!lazer finish and release as a major update for osu! ?

There is no estimate for this.

> Question 2. Will the conversion and fun mods from osu!lazer be ranked? Because I see some of them make the game harder or easier

There is no such thing as "ranked mods" in osu!(lazer) anymore.

> Issue 1: When playing osu!lazer, I often got lag spike both in the main menu and during the game, game freeze but music not, after that the game resume but I will miss a lot or fail because I can't see click the note during freeze

Likely known (https://github.com/ppy/osu/issues/11800) but we need videos of this to confirm.

> Issue 2: If the game is in full-screen mode and alt-tabbed away from the game then coming back, osu!lazer will freeze, music stop playing, then I had to close the game manually and restarted it in order to play the game

Likely also known (https://github.com/ppy/osu/issues/10157). If you're on an nvidia graphics card and have shadowplay enabled then please disable it.

> Issue 3: Some skin cannot be imported and rolled back in osu!lazer

We cannot help you with this without a link to the "skins" in question.
